web_server/server.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
I'm not very familiar with the framework for these demos.
I assume this file is needed/recommended. Just to confirm that this is the case.

Reply:
It's needed to run a local dev environment  - documented in the test repo. I missed it here, so copied it in.